xrpl/wallet/main.py

```diff
@@ -216,9 +215,11 @@ def from_secret_numbers(
             master_address: Include if a Wallet uses a Regular Key Pair. It must be
                 the master address of the account. The default is `None`.
             algorithm: The digital signature algorithm to generate an address for.
-                The default is `SECP256K1
-                <https://xrpl.org/cryptographic-keys.html#secp256k1-key-derivation>`_
-                (XUMM standard as of December 2022).
+                The default is ED25519. Docs:
+                https://xrpl.org/docs/concepts/accounts/cryptographic-keys#ed25519-key-derivation
```

Comment:
Let's keep the link format as it was before.

Reply:
Ok, fixed in the latest commit.